### PR TITLE
Refactor courses & assignments

### PR DESCRIPTION
**Pull request by Felienne**
- Because we now have adventures to manage courses, old code loading assignments can be removed, in the code and in the templates.

- The Space EU course no longer needs maintenance (now simple redirects to the Hedy course)

- Online masters now uses new "custom" code, set a course to custom and provide a list of adventures and just those adventures will be shown.

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -693,17 +683,16 @@ def space_eu(level, step):
...
     g.prefix = '/space_eu'
 
-    adventure_assignments = load_adventure_assignments_per_level(g.lang, level)
+    adventures = load_adventures_per_level(g.lang, level)
 
-    return hedyweb.render_assignment_editor(
+    return hedyweb.render_code_editor_with_tabs(
         request=request,
-        course=SPACE_EU_COURSE[g.lang],
+        course=HEDY_COURSE[g.lang],
```
Comment (rix0rrr) [inline app.py:690]: What is the goal for the `space_eu` course? The PR description said it would redirect to the regular Hedy course. If you want to do that,  you could replace this whole block with something like:

```py
    return redirect(url_for('index', level=level, step=step), code=302)
```

It would cut down on unnecessary code if we did that.

File: coursedata/course/online_masters/nl.yaml [hunk 2]
```diff
@@ -1,150 +1,12 @@
 ---
 course:
-  -
-    level: "1"
-    assignments:
-      - step: "1"
-        prompt: |
-          <a href="https://youtu.be/LKnQvUvCRqM" target="_blank">Klik hier om de uitlegvideo te bekijken.</a>
```
Comment (rix0rrr) [inline coursedata/course/online_masters/nl.yaml:8]: Are you okay losing the YouTube video links? I don't think those are in the regular adventures, are they?

File: templates/incl-editor-and-output.html [hunk 3]
```diff
@@ -1,11 +1,11 @@
 <div class="grid grid-cols-1 lg:grid-cols-2 gap-4 mt-5">
     <div>
       <div class="w-full h-64 flex flex-col">
-        <pre id="editor"
-          style="background: #272822; color: white"
+        <div id="editor"
```
Comment (rix0rrr) [inline templates/incl-editor-and-output.html:4]: Why this change?

**Review by rix0rrr — APPROVED** [review]

File: coursedata/course/online_masters/nl.yaml (hunk 2 above)
Comment (Felienne) [reply coursedata/course/online_masters/nl.yaml:8]: Yeah we can live without them (they are really old by now anyway)
